zanata-war/src/main/java/org/zanata/service/impl/DocumentServiceImpl.java

```diff
@@ -265,8 +265,10 @@ private void processWebHookDocumentMilestoneEvent(
             WordStatistic oldStats = StatisticsUtil.copyWordStatistic(stats);
             if(oldStats != null) {
 
-                for(Map.Entry<ContentState, Integer> entry: event.getWordDeltasByState().entrySet()) {
-                    oldStats.increment(entry.getKey(), entry.getValue());
+                for (Map.Entry<ContentState, Long> entry : event
+                        .getWordDeltasByState().entrySet()) {
+                    oldStats.increment(entry.getKey(),
+                            entry.getValue().intValue());
```

Comment:
Perhaps we don't need to push the int->long conversion into AbstractStatistic, but we should have a note about the down-conversion here (calling `intValue()` on a Long).